sdk/include/psdk/shellapi.h

```diff
@@ -669,8 +674,7 @@ typedef LPSHNAMEMAPPINGW LPSHNAMEMAPPING;
 #define SHEmptyRecycleBin SHEmptyRecycleBinW
 #define SHGetNewLinkInfo SHGetNewLinkInfoW
 #define DoEnvironmentSubst DoEnvironmentSubstW
-
-#else
+#else /* UNICODE */
```

Comment:
Adding some spacing because we don't otherwise see where the blocks start and end...
```suggestion

#else /* UNICODE */

```

Reply:
Added.